gen/wrap.jl

```diff
@@ -136,14 +142,19 @@ function rewriter(x::Expr)
             false
         end
 
-        cc2 = if rettype == :Cstring
-            :(unsafe_string($cc))
-        elseif rettype == :(Ptr{Cstring})
-            :(unsafe_loadstringlist($cc))
-        elseif isptrvoid
-            :(failsafe($cc))
+        cc2 = if occursin(r"^cpl.*error", String(f2))
+            # since we call these in aftercare, no aftercare to prevent calling itself
```

Comment:
This feels very meta. Can it be reworded?

Reply:
Ha what would be better? Without this you basically get StackOverflow errors, since `cplgetlasterrortype` calls `aftercare` which calls `cplgetlasterrortype` and so forth.

Reply:
The comments was for the comment! It just needs a small introduction for aftercare.
```julia
# We will take care of gdal error states and memory freeing (for now) by calling `aftercare`, except for `*error*` functions, as these will be called by `aftercare` itself, so we prevent a loop.
```